index.html

```diff
@@ -1935,13 +1935,35 @@ <h4 class="text-warning">We believe that cyclists and triathletes should be able
                             </div>
                         </section>
                         
-                        <section id="section-more" class="section appear clearfix">
-                            <div class="container">
-                                <h1>More</h1>
-                                <i class="fa fa-twitter"></i> Twitter

```

Comment:
To display [`fa-twitter`](http://fontawesome.io/icon/twitter/) icon, we need to setup [Font Awesome](http://fontawesome.io/).